### PR TITLE
jhsdb: add page

### PR DESCRIPTION
**Pull request by WhileLoop**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by WhileLoop** [comment]
Resolves https://github.com/tldr-pages/tldr/issues/4177

File: pages/common/jhsdb.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+> Attach to a Java process or launch a postmortem debugger to analyze the content of a core dump from a crashed Java Virtual Machine (JVM).
+
+- Using pid to connect to a hung process:
+
+`jhsdb jstack --pid {{pid}}`
+
+- Open a core dump for analysis:
+
+`jhsdb clhsdb --core {{./core}} --exe {{/path/to/jdk/bin/java}}`
```
Comment (blueskyson) [inline pages/common/jhsdb.md:11]:
```suggestion
`jhsdb clhsdb --core {{path/to/core_dump}} --exe {{/path/to/jdk/bin/java}}`
```

File: pages/common/jhsdb.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Start a debug server:
+
+`jhsdb debugd --pid {{pid}} --serverid {{server_id}}`
```
Comment (blueskyson) [inline pages/common/jhsdb.md:15]: How to determine a server id? Is it a hostname or an address?

File: pages/common/jhsdb.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# jhsdb
+
+> Attach to a Java process or launch a postmortem debugger to analyze the content of a core dump from a crashed Java Virtual Machine (JVM).
```
Comment (blueskyson) [inline pages/common/jhsdb.md:3]:
```suggestion
> Attach to a Java process or launch a postmortem debugger to analyze the core dump from a crashed Java Virtual Machine (JVM).
> More information: <https://docs.oracle.com/en/java/javase/16/docs/specs/man/jhsdb.html>.
```

**Review by blueskyson — COMMENTED** [review]
Thank you for contribution. Here are some suggestions.

File: pages/common/jhsdb.md (hunk 2 above)
Comment (WhileLoop) [reply pages/common/jhsdb.md:15]: Docs say: An optional unique ID to use when multiple debug servers are running on the same remote host. Changing to {{optional_unique_id}}.

**Comment by WhileLoop** [comment]
@blueskyson Updated per your suggestions.

File: pages/common/jhsdb.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# jhsdb
+
+> Attach to a Java process or launch a postmortem debugger to analyze the core dump from a crashed Java Virtual Machine.
+
```
Comment (CleanMachine1) [inline pages/common/jhsdb.md:4]:
```suggestion
More information: <https://manned.org/jhsdb>.

```

File: pages/common/jhsdb.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+- Using pid to connect to a hung process:
```
Comment (CleanMachine1) [inline pages/common/jhsdb.md:5]:
```suggestion
- Use a PID to connect to a hung process:
```

**Comment by WhileLoop** [comment]
@CleanMachine1 Suggestions applied.

File: pages/common/jhsdb.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+> Attach to a Java process or launch a postmortem debugger to analyze the core dump from a crashed Java Virtual Machine.
+> More information: <https://manned.org/jhsdb>.
+
+- Use a PID to connect to a hung process:
+
+`jhsdb jstack --pid {{pid}}`
+
+- Open a core dump for analysis:
+
+`jhsdb clhsdb --core {{path/to/core_dump}} --exe {{/path/to/jdk/bin/java}}`
```
Comment (marchersimon) [inline pages/common/jhsdb.md:12]: Does the `--exe` path have to be absolute (starting at `/`)?

File: pages/common/jhsdb.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# jhsdb
+
+> Attach to a Java process or launch a postmortem debugger to analyze the core dump from a crashed Java Virtual Machine.
+> More information: <https://manned.org/jhsdb>.
+
+- Use a PID to connect to a hung process:
+
+`jhsdb jstack --pid {{pid}}`
+
+- Open a core dump for analysis:
```
Comment (marchersimon) [inline pages/common/jhsdb.md:10]: What do you mean by `open`?

File: pages/common/jhsdb.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+`jhsdb clhsdb --core {{path/to/core_dump}} --exe {{/path/to/jdk/bin/java}}`
+
+- Start a debug server:
+
+`jhsdb debugd --pid {{pid}} --serverid {{optional_unique_id}}`
+
+- Connect to process in clhsdb mode:
```
Comment (marchersimon) [inline pages/common/jhsdb.md:18]:
```suggestion
- Connect to a process in interactive debug mode:
```

File: pages/common/jhsdb.md (hunk 7 above)
Comment (WhileLoop) [reply pages/common/jhsdb.md:10]: Is `Open a core dump in interactive debug mode:` better?

Comment (marchersimon) [reply pages/common/jhsdb.md]: Yes, the `interactive` is important here. Otherwise it sounds like it is opened and then the command finishes.

**Review by marchersimon — APPROVED** [review]
Thanks @WhileLoop

**Review by CleanMachine1 — APPROVED** [review]
Merge either @marchersimon or @navarroaxel :+1:

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**